server/streamable_http_test.go

```
@@ -0,0 +1,677 @@
+		},
+	},
+}
+
+func addSSETool(mcpServer *MCPServer) {
+	mcpServer.AddTool(mcp.Tool{
+		Name: "sseTool",
+	}, func(ctx context.Context, req mcp.CallToolRequest) (*mcp.CallToolResult, error) {
+		// Send notification to client
+		server := ServerFromContext(ctx)
+		for i := 0; i < 10; i++ {
+			server.SendNotificationToClient(ctx, "test/notification", map[string]any{
```

golangci-lint (1.64.8)

Error return value of `server.SendNotificationToClient` is not checked

(errcheck)